frontend_app_thread/src/container/Thread.jsx

```diff
@@ -48,8 +48,9 @@ export class Thread extends React.Component {
       breadcrumbsList: [],
       isVisible: true,
       config: param.config,
-      loggedUser: param.loggedUser,
       content: param.content,
+      customTimeline: [],
```

Comment:
Is this state used?

Reply:
Solved in [fd85e03](https://github.com/tracim/tracim/pull/5846/commits/fd85e03f05599f33b4ee4f2e616fd52f78605da7).